src/Advent/Utils.hs

```diff
@@ -22,6 +26,6 @@ combinations n xs =
       ys <- combinations (pred n) xs'
   ]
 
-fromRightOrError' :: Show a => Either a b -> b
-fromRightOrError' (Left x) = error (show x)
-fromRightOrError' (Right x) = x
+fromRightOrShowError :: Show a => Either a b -> b
+fromRightOrShowError (Left x) = error (show x)
+fromRightOrShowError (Right x) = x
```

Comment:
I was lookin' for a name for this thing.  This is inspired by [Debug.Trace.traceShow](https://hackage.haskell.org/package/base-4.14.0.0/docs/Debug-Trace.html#v:traceShow)